src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -4,87 +4,95 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing iOS project.
 ---
 
-TODO(jmagman): add the pre-built framework alternative and review.
+Flutter can be embedded into your existing iOS application in a piecemeal fashion as
+frameworks or libraries.
 
-## Integration steps
+## Create a Flutter module
 
-To integrate a Flutter module into your app, use the following steps:
+In order to embed Flutter into your existing application, first create a Flutter module.
 
- 1. [Create Flutter Module](#create-a-flutter-module)
-   f Creates the Flutter project that is hosted in the existing app.
- 1. [Add Module Dependency in Existing
-    App](#make-the-host-app-depend-on-the-flutter-module)
-    Sets up Xcode to include your Flutter project into the build of your
-    existing app.
+From the command line, run:
 
-## Create a Flutter module
+```terminal
+cd some/path/
+flutter create --template module my_flutter
+```
 
-Flutter projects created using `flutter create xxx` include simple host apps for
-your Flutter/Dart code (a single-ViewController iOS host). You can modify these
-host apps to suit your needs and build from there.
+TODO(jmagman): IDE instructions
 
-If you're starting off with an *existing* host app for either platform, you'll
-likely want to include your Flutter project in that app as some form of library
-instead.
+This creates a Flutter module project at `some/path/my_flutter/`.
 
-In order to embed Flutter as a library, use the Flutter module template.
-Executing `flutter create -t module xxx` produces a Flutter project with a
-CocoaPods pod designed for consumption by your existing host app.
+In that directory you can run the same `flutter` commands you would
+in any other Flutter project, like `flutter run --debug` or `flutter build ios`.
+You can also run in [Android Studio/IntelliJ][] or [VS Code][] with the Flutter and Dart plugins.
+This will run a single-view example version of your module without being
+embedded in your existing application. This can be useful for testing
+quick, incremental changes with hot reload.
```

Comment:
this makes it sound like you can't hot reload if you embedded into your existing app